ajax/libs/jodit/package.json

```diff
@@ -10,18 +10,36 @@
     "rich",
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
   "author": "Chupurnov <chupurnov@gmail.com> (http://xdsoft.net/)",
   "repository": {
     "type": "git",
     "url": "https://github.com/xdan/jodit.git"
   },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xdan/jodit.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "jodit.min.js",
+          "jodit.min.css",
+          "jodit.es2018.min.js",
+          "jodit.es2018.min.css"
+        ]
+      }
+    ]
+  },
   "npmName": "jodit",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "build",
       "files": [
-        "*jodit*.+(css|js|map)"
+        "jodit.min.js",
```

Comment:
Why only include the minified files, would it not be sensible to also include the unminified files as well?

Reply:
Added *jodit*.+(css|js|map)